installer-howto/installer-howto.rst

```diff
@@ -13,11 +13,11 @@ Specifically with our fledgling move / rebase on openSUSE Leap15.2 our thanks go
 and the larger community for the extreme generosity we all benefit from in a myriad of seen and unseen ways.
 
 **Please note: Rockstor is in no way affiliated with openSUSE / SuSE and does not wish, under any circumstances, to misrepresent these organisations.**
-*Any errors you encounter are most likely our fault and we try, on a current volunteer basis, to support our users via our* `friendly forum <https://forum.rockstor.com/.>`_.
+*Any errors you encounter are most likely our fault and we try, on a current volunteer basis, to support our users via our* `friendly forum <https://forum.rockstor.com/>`_.
 
 It is hoped that Rockstor’s ongoing small contribution back to the larger community is of value,
 and if this is so for you please consider either subscribing to our Stable Channel updates or taking part in our developer orientated Testing Channel.
-Details of these Rockstor package update services are available in our `Official Docs <http://rockstor.com/docs/>`_ subsection `Update Channels <http://rockstor.com/docs/update-channels/update_channels.html>`_.
+Details of these Rockstor package update services are available in our `Official Docs <https://rockstor.com/docs/>`_ subsection `Update Channels <https://rockstor.com/docs/update-channels/update_channels.html>`_.
```

Comment:
@FroggyFlox I'm thinking that in these, self referencing links we use internal links as we are referencing not to the internet but to other parts of this same repo. This way we side step the above http/https issue with regard to the python testing locally and make for a more portable testing setup. Not sure how the rockstor.com/docs would work here but presumably it would be our doc index page, which in turn could have an internal link.

Reply:
Agreed, I switched those links to internal references and took the opportunity to reformat the whole page to lines of 80 characters max length.
Thanks!